### ISSUE TITLE
Double quotes removed from host_integrations config

### ISSUE THREAD

**Issue by maximusfloydus** [newrelic/infrastructure-agent-chef#57]
[infra_helper](https://github.com/newrelic/infrastructure-agent-chef/blob/16c8aad771cf7992accd3a545e1de8cffe486f42/libraries/infra_helper.rb#L31 ) is removing all double quotes from host_integrations configuration which is causing an issues with configuring [nri-rabbitmq](https://github.com/newrelic/nri-rabbitmq).

Example config:
```
node.default['newrelic_infra']['host_integrations']['config']['nri-rabbitmq']['integration_name'] = 'com.newrelic.rabbitmq'
  node.default['newrelic_infra']['host_integrations']['config']['nri-rabbitmq']['instances'] = [
    {
      name: 'localhost',
      command: 'all'
      arguments: {
        username: 'monitor',
        vhosts: '["/"]',
        exchanges_regexes: '[".*"]',
        queues_regexes: '["zero"]'
      }
    }
  ]
```
This will produce an output file:
```
integration_name: com.newrelic.rabbitmq
instances:
  - name: localhost
    command: all
    arguments:
      username: monitor
      vhosts: '[/]'
      exchanges_regexes: '[.*]'
      queues_regexes: '[zero]'
```
The newrelic-infra agent then throws errors about the strings in the arrays for vhosts
```
newrelic-infra: time="2019-05-20T08:31:58-05:00" level=error msg="executing data source" data prefix=config/rabbitmq error="exit status 255" plugin name=localhost stderr="[ERR] Error parsing arguments [Vhosts]: invalid character '/' looking for beginning of value\n"
```

[rabbitmq-monitoring-integration#config](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/rabbitmq-monitoring-integration#config) shows the format should be `'["myQueue1","myQueue2"]'`

Not sure if this a defect for this cookbook or how nri-rabbitmq requires configuration.

**Comment by alejandrodnm** [comment]
Hi @maximusfloydus I made a PR #63 would you mind taking a look at it and let me know if this would work for you?

**Comment by alejandrodnm** [comment]
this has been released in [0.11.0](https://github.com/newrelic/infrastructure-agent-chef/releases/tag/0.11.0)
